igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2015 Netflix, Inc.
+ * Copyright (c) 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
[No need](http://www.apache.org/dev/apply-license.html#contributor-copyright) to append Copyrights on these existing files.